lib/rspec/core/configuration.rb

```diff
@@ -828,6 +828,18 @@ def profile_examples
       end
 
       # @private
+      def profiler
+        @profiler ||=
+          begin
+            if value_for(:profile_examples) { @profile_examples }
```

Comment:
Can't this use the `profile_examples` method instead of repeating its logic to get a value?